pybots/web/admin_index_view.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+
+from flask.helpers import url_for
+from flask.templating import render_template
+from flask.views import MethodView
+from werkzeug.utils import redirect
+
+from pybots.configurations.configuration_provider import configuration_provider
+from pybots.game.game_controller import game_controller
+from pybots.views.utils import form_to_kwargs
+from pybots.web.forms import ConfigurationForm
+from pybots.web.game_detail_view import GameDetailView
+
+
+class AdminIndexView(MethodView):
+    decorators = [form_to_kwargs]
+
+    def get(self):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/thejoeejoee/pybots-server/file/2266725736/issues/source?bid=3184258&fileBranchId=3184263#l18)